lib/vmdb/plugins.rb

```diff
@@ -4,29 +4,53 @@ class Plugins
 
     attr_reader :vmdb_plugins
     attr_reader :registered_automate_domains
+    attr_reader :registered_provider_plugins
 
     def initialize
       @registered_automate_domains = []
+      @registered_provider_plugins = []
       @vmdb_plugins = []
+
+      Rails.application.railties.each do |railtie|
+        next unless railtie.class.name.start_with?("ManageIQ::Providers::") || railtie.try(:vmdb_plugin?)
```

Comment:
Can the `registered_provider_plugins` logic be split into a stand alone PR, independent of EmsEvents and moving the initializer logic?